framework/src/st_things/things_stack/cloud/cloud_manager.c

```diff
@@ -310,7 +311,7 @@ static int set_def_cloud_info(es_cloud_signup_s *cloud_info, const char *cloud_a
 		if (strchr(t_domain, ':') != NULL) {
 			ip_port = strchr(t_domain, ':');
 			if (ip_port) {
-				*ip_port = NULL;
+				ip_port = NULL;
```

Comment:
you can remove line 312  ~ 315 line.

Reply:
In this PR, I handle build time warnings. Remove line issue, I will check in another PR.